x-pack/legacy/plugins/graph/public/components/search_bar.tsx

```diff
@@ -12,7 +12,10 @@ import { connect } from 'react-redux';
 import { fromKueryExpression, toElasticsearchQuery } from '@kbn/es-query';
 import { IDataPluginServices, Query } from 'src/plugins/data/public';
 import { IndexPatternSavedObject, IndexPatternProvider } from '../types';
-import { QueryBarInput, IndexPattern } from '../../../../../../src/legacy/core_plugins/data/public';
+import {
+  QueryStringInput,
+  IndexPattern,
```

Comment:
Should `IndexPattern` be switched to `IIndexPattern` here?

Reply:
You're right, though I try to make fewer changes to other plugins

Reply:
Yes, but i'm trying to make fewer changes to other plugins